core/cmd/shell_local.go

```diff
@@ -474,18 +469,6 @@ func (s *Shell) runNode(c *cli.Context) error {
 		}
 	}
 
-	if s.Config.Capabilities().Peering().Enabled() {
```

Comment:
woah - what are the implications of removing this block?

Reply:
I think I rebased it incorrectly, fixing that